src/interfaces/IOptionSettlementEngine.sol

```diff
@@ -438,6 +438,18 @@ interface IOptionSettlementEngine {
     //  Protocol Admin
     //////////////////////////////////////////////////////////////*/
 
+    /**
+     * @notice Check if the protocol fee switch is enabled.
+     * @return Whether or not the protocol fee switch is enabled.
+     */
+    function isFeeSwitchEnabled() external view returns (bool);
```

Comment:
Why not just `feeSwitch returns (bool enabled)`

Reply:
Sure, that works.